Fabric.Identity.API/scripts/Register.ps1

```diff
@@ -684,4 +684,7 @@ Invoke-RegisterClients -clients $clients -identityServiceUrl $identityServiceUrl
 $authorization = Get-RolesAndPermissionsToRegister -registrationConfig $registrationSettings
 Invoke-RegisterSharedRolesAndPermissions -rolesAndPermissions $authorization -identityServiceUrl $identityServiceUrl -accessToken $accessToken
 
-Read-Host -Prompt "Registration complete, press Enter to exit"
\ No newline at end of file
+# Upgrade users with EDWAdmin role to dosadmin role
+Invoke-Sqlcmd -InputFile "upgrade-admin-role.sql"
```

Comment:
Is it possible to write a message to the console so the user running the script has some feedback as to what is happening?

Reply:
Definitely. Great suggestion.